### PR TITLE
Do not call `chown` to config files

### PR DESCRIPTION
**Pull request by reasonerjt**
This commit fixes a recently discovered issue on Kubernetes #4496
It make necessary to avoid calling `chown` to config files during the
bootstrap of the containers.

### CODE REVIEW

File: make/prepare [hunk 1]
```diff
@@ -173,6 +182,10 @@ def delfile(src):
             itemsrc=os.path.join(src,item)
             delfile(itemsrc)
 
+if not os.path.exists(JOB_LOG_DIR):
+    os.makedirs(JOB_LOG_DIR)
+mark_file(JOB_LOG_DIR, mode=0o655)
```
Comment (jessehu) [inline make/prepare]: Should the mode be 755?  655 is seldomly seen.

File: make/photon/registry/entrypoint.sh [hunk 2]
```diff
@@ -2,12 +2,11 @@
 
 set -e
 
-if [ -d /etc/registry ]; then
-    chown 10000:10000 -R /etc/registry
-fi
 if [ -d /var/lib/registry ]; then
+    # For CI testng.
```
Comment (jessehu) [inline make/photon/registry/entrypoint.sh]: please add detail description for this chown, and 'testng' is a typo.

File: make/prepare (hunk 1 above)
Comment (reasonerjt) [reply make/prepare]: You are right...

File: make/photon/registry/entrypoint.sh (hunk 2 above)
Comment (reasonerjt) [reply make/photon/registry/entrypoint.sh]: yep, will do.

**Comment by cristifalcas** [comment]
Can we have new images with those changes?

**Pull request merged**